metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
pyOpenSSL does not have a function to generate dhparameters. Using subprocess kept silently failing during charm deployment (not in this charm, but I tested it in the sojobo-api charm)